readme.md

```diff
@@ -87,6 +87,16 @@ Type: `number`
 
 The minimum amount of bytes needed to detect a file type. Currently, it's 4100 bytes, but it can change, so don't hardcode it.
 
+### fileType.stream(readableStream)
+
+Detect the file type of a readable stream.
+
+Returns a `Promise` which resolves to the original readable stream argument, but with an added `fileType` property, which is an object like the one returned from `fileType()`.
+
```

Comment:
Can you include a quick example? For example: https://github.com/sindresorhus/file-type/pull/187/files#diff-1dd241c4cd3fd1dd89c570cee98b79ddR230

Reply:
Yup for sure